config/crd/kustomization.yaml

```diff
@@ -6,3 +6,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 patches:
 - path: patches/webhook_in_dynakubes.yaml
+- path: patches/webhook_in_edgeconnects.yaml
```

Comment:
I may be wrong here but is this file missing? Because if I want to apply ec I get the message that this file is missing.

Reply:
my fault, sorry, added